show/main.py

```diff
@@ -1133,6 +1120,215 @@ def ntp(verbose):
     print(tabulate(ntp_dict, headers=list(ntp_dict.keys()), tablefmt="simple", stralign='left', missingval=""))
 
 
+# 'snmp' subcommand ("show runningconfiguration snmp")
+@runningconfiguration.group("snmp", invoke_without_command=True)
+@clicommon.pass_db
+@click.pass_context
+def snmp(ctx, db):
+    """Show SNMP running configuration"""
+    if ctx.invoked_subcommand is None:
+       show_run_snmp_global(db.cfgdb)
+
+
+# '("show runningconfiguration snmp community")
```

Comment:
>'(" [](http://example.com/codeflow?start=2&length=3)

Could you remove the extra marks in comment?

Reply:
I'll remove the "'"